scipy-1.0/paper.tex

```diff
@@ -1156,9 +1098,14 @@ \section*{Data Availability}
 
 \bibliography{references}
 
+\section*{Acknowledgements}
+
+We thank everyone who has contributed to SciPy 1.0, from those who have posted one comment about an issue through those who have made several small patches and beyond!
+
 \section*{Author Contributions Statement}
 
-All authors have contributed code to SciPy, and all authors reviewed the manuscript. In the author list, the names of authors who contributed text to the manuscript are indicated with $\dagger$, and the primary manuscript editors are denoted with $\ddagger$.
+P.V. is SciPy's Benevolent Dictator for Life, R.G. is the Chair of the SciPy Steering Council, and together they have led the development of SciPy for over 10 years. T.E.O., E.J., and P.P. created SciPy. T.J.R. and M.H. composed the manuscript with input from others. Other named authors are SciPy Core Developers. All authors have contributed significant code, documentation, and/or expertise to the SciPy project. All authors reviewed the manuscript.
```

Comment:
I would also change the contribution statement to mark Core Dev and Paper contributors similarly to first version from @mdhaber 
```suggestion
P.V. is SciPy's Benevolent Dictator for Life, R.G. is the Chair of the SciPy Steering Council, and together they have led the development of SciPy for over 10 years. T.E.O., E.J., and P.P. created SciPy. T.J.R. and M.H. composed the manuscript with input from authors indicated with $\ddagger$. Authors indicated with $\dagger$ are SciPy Core Developers. All authors have contributed significant code, documentation, and/or expertise to the SciPy project. All authors reviewed the manuscript.
```